index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<div class="container">
+	<div>
+		<article class="cardbox">
+	  		<header>
+	  			<h1>Beach Front Property - Nebraska</h1>
+	  		</header>
+	  			<section>	
+	  			  	<img src="beach.jpg" alt="">
+	  			  	<p class="P1">Description Text Description Text Description Text Description Text Description Text Description TextDescription Text Description TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription Text
+	  			  	</p>
+	  				<p class="avail">Available</p>
+	  			</section>
+	  		<section class = "sec2">	
+		  		<h2>Specifications</h2>
+		  			<p class="bold">Size: Huge</p>
+		  			<p class="bold">Weight: Weightless</p>
+		  			<footer>Specifications valid until you view property in person</footer>
+		  	</section>
+		  	<section class = "sec3">
+		  		<h3>Pricing</h3>
+		  		<p class="bold">1: Cord of Wood</p>
+		  		<p class="bold">2: A chunk of Frankincense</p>
+	  			<p class="bold">3: Barrels of Wheat</p>
+	  		</section>
+		</article>
+</div>
+<div>
+	<article class="cardbox">
+  		<header>
+  			<h1>Beach Front Property - Iowa</h1>
+  		</header>
+  			<section>	
+  			  	<img src="beach.jpg" alt="">
+  			  	<p class="P1">Description Text Description Text Description Text Description Text Description Text Description TextDescription Text Description TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription TextDescription Text
+  			  	</p>
+  				<p class="avail">Available</p>
+  			</section>
+	  		<section class = "sec2">	
+		  		<h2>Specifications</h2>
+		  			<p class="bold">Size: Huge</p>
+		  			<p class="bold">Weight: Weightless</p>
+		  			<footer>Specifications valid until you view property in person</footer>
+		  	</section>
+	  	<section class = "sec3">
+		  	<h3>Pricing</h3>
+		  		<p class="bold">1: Cord of Wood</p>
+		  		<p class="bold">2: A chunk of Frankincense</p>
+	  			<p class="bold">3: Barrels of Wheat</p>
+	  	</section>
+	</article>
+</div>
+<div>
+	<article class="cardbox">
```

Comment:
Now that I look at this more closely, I would recommend not having this second container at all. IF you look at the requirements, it states that each card should take up 30% of the window width. If you adjust your CSS to make that the case, then the fourth card will naturally jump below the other 3, even if everything is in the same container. That will be a much more reliable layout, and more like the kind of markup you'll write once we are dynamically adding items to the DOM, rather than hard coding them like this.